### PR TITLE
Release 3.0.7

### PR DESCRIPTION
**Pull request by rbnvrw**

### CODE REVIEW

**Comment by rbnvrw** [comment]
@jakirkham  Could you please help with this PR? the Appveyor build repeatedly fails and I can't figure out what's wrong.

**Comment by jakirkham** [comment]
@conda-forge-admin, please rerender.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -14,6 +14,7 @@ source:
 build:
   number: 0
   script: python setup.py install --single-version-externally-managed --record record.txt
+  noarch: python
```
Comment (jakirkham) [inline recipe/meta.yaml:17]: When adding/removing `noarch`, one should also re-render.

**Comment by jakirkham** [comment]
FTR that command is available to you to run as well.

**Pull request merged**